shared/naturalcrit/splitPane/splitPane.jsx

```diff
@@ -26,6 +25,18 @@ const SplitPane = createClass({
 				size : paneSize
 			});
 		}
+		window.addEventListener('resize', this.resetSize);
```

Comment:
This should happen in the `componentDidMount` step instead.

Reply:
It was - however the lack of a blank line between `getInitialState` and `componentDidMount` was causing the GitHub diff tool to only show the top of the block of text, which was `getInitialState`.

Reply:
Oh hah. I'm dumb. Viewing this on mobile makes it even worse.